modules/users/client/views/profile/profile-edit-account.client.view.html

```diff
@@ -137,6 +137,44 @@
 </div>
 <!-- /Password -->
 
+<!-- Notifications -->
+<div class="panel panel-default">
+  <div class="panel-heading">
+    Notifications
+  </div>
+  <div class="panel-body">
+
+    <form autocomplete="off">
+      <div class="form-horizontal">
+
+        <div class="form-group">
+          <label class="col-sm-3 text-right control-label">Push</label>
+          <div class="col-sm-9">
+
+            <div class="checkbox">
+              <label>
+                <input type="checkbox"
+                       name="browser"
+                       ng-model="profileEditAccount.pushEnabled"
+                       ng-disabled="profileEditAccount.pushIsDisabled()"
+                       ng-change="profileEditAccount.pushUpdate()">
+                Current web browser
+              </label>
+            </div>
+
+            <p class="help-block" ng-if="profileEditAccount.push.isSupported"><small>This will send notifications even when the website is closed.</small></p>
+            <p class="help-block" ng-if="!profileEditAccount.push.isSupported"><small>Your browser does not support push notifications.</small></p>
```

Comment:
I did it that way so it communicates: _if you use another browser then you can get push notifications_

It also helps to keep a consistent interface, so users don't get confused wondering where the option went when they had it available on another browser.

I don't feel _that_ strongly about it though. Happy to hide it all if unsupported if you prefer.